lib/index.js

```diff
@@ -78,45 +80,50 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
         price,
         updateFeed,
         updateFiat,
+        hideMenus = false,
         refreshPrice
       } = this.props;
 
       const Ticker = () => (
-        <div className={'col'} style={{height: '100%', lineHeight: '200%'}}>
+        <div className={'col'} style={{ height: '100%', lineHeight: '200%' }}>
           <Text>
-            {price.crypto +
-              ' ' +
+            {'1 ' + price.crypto +
```

Comment:
Do you think it's not obvious to crypto users that the displayed fiat amount equals `1` of the crypto? :-D

Reply:
I don't see it so much as as a crypto thing as just a exchange rate convention. Just reads a little clearer to me, like when you're at an airport exchanging money.

Reply:
Ah hmm ok I'm just not in love with how it looks, takes up a little bit extra space for what I see as redundant. I don't see the `1 BTC =` on any other Bitcoin website

Reply:
That's fair. At a certain level it is just a personal preference thing I guess and this is your widget :).

For context though, in "normal" exchanges this is what I've seen (and how about that headline!) from https://money.cnn.com/data/currencies/:
<img width="632" alt="screen shot 2019-01-14 at 7 59 36 pm" src="https://user-images.githubusercontent.com/4344978/51153811-2dab9c00-1837-11e9-9366-080638f8c107.png">

I find it helpful when you're using a UI that can be a number of different combos. But again, personal preference! Happy to change if you prefer.